### PR TITLE
Set version to 1.3.3

### PR DESCRIPTION
**Pull request by rikhuijzer**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 This is a log of major changes in Gadfly between releases. It is not exhaustive.
 Each release typically has a number of minor bug fixes beyond what is listed here.
 
+# Version 1.3.3
+
+ * Add support for CategoricalArrays 0.10 (#1537)
+ * Add support for Showoff 1.0 (#1522)
+ * Make Gadfly "relocatable" by not loading files at runtime (#1530)
+ * Disable CI testing for Julia < 1.4 (#1525)
```
Comment (Mattriks) [inline NEWS.md:9]: The issue numbers should be in reverse order.  Also the main idea in NEWS.md was to document grammar of graphics features, not housekeeping issues.

Comment (rikhuijzer) [reply NEWS.md]: Good point. My bad. I've reversed the order and only kept the version bumps. That's a trade-off between documenting grammar of graphics features and not having an empty NEWS entry, which, I guess, could be a bit confusing for the reader.

**Comment by Mattriks** [comment]
I'll merge this, & I'll fix the PR numbers in the news.md myself sometime ....

**Comment by rikhuijzer** [comment]
> I'll merge this, & I'll fix the PR numbers in the news.md myself sometime ....

Uhm...

- The numbers are linked to the correct PRs, so that isn't broken.
- The order is now ascending chronological like you asked, so that isn't broken.
- If it is about only showing grammar of graphics features, then you could edit that in three clicks in the GitHub interface.

I try to learn from comments to make better PRs in the future but am unsure what to do with your comment about the PR numbers which need to be fixed.

**Comment by rikhuijzer** [comment]
> I'll merge this, & I'll fix the PR numbers in the news.md myself sometime ....

Maybe, the template at <https://github.com/JuliaWeb/HTTP.jl/blob/master/CHANGELOG.md> is nice

**Pull request merged**